src/dtl/dyad_dtl.h

```diff
@@ -0,0 +1,20 @@
+#ifndef DYAD_DTL_H
+#define DYAD_DTL_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+    
+enum dyad_dtl_mode {
+    DYAD_DTL_UCX = 0,
+    DYAD_DTL_FLUX_RPC = 1,
```

Comment:
add something like DYAD_DTL_END without comma at the end

Reply:
Why? I'm not sure why we'd need a `DYAD_DTL_END`. The `dyad_dtl_mode` enum is used to select which underlying implementation of the DTL we're going to use.

Reply:
Thanks for clarifying over Slack. I've added this